pkg/controller/handler.go

```diff
@@ -235,6 +273,23 @@ func (h *handler) CreateServiceBroker(in *servicecatalog.Broker) (*servicecatalo
 	return h.apiClient.Brokers().Update(in)
 }
 
+func (h *handler) unbind(b *servicecatalog.Binding) error {
+	inst, err := instanceForBinding(h.apiClient, b)
+	if err != nil {
+		return err
+	}
+	sc, err := serviceClassForInstance(h.apiClient, inst)
+	if err != nil {
+		return nil
+	}
+	broker, err := brokerForServiceClass(h.apiClient, sc)
```

Comment:
These three calls still seem like a weird concretization of what we're asking for. I realize it's because they're not linked up. Is this an API we should put on the client to do lookups?

It's very obvious what they are doing, so I give credit to the names. The software guy in me goes, 'hmn'. 

No changes needed here. Discussion only.